<launch>
    <node pkg="turn" type="turn_action" name="turn_action" output="screen">
	<param name="odom_topic" value="/my_odom" />
	<param name="storage" value="$(find navigation)/storage"/>
    </node>
</launch>
